Vagrantfile

```diff
@@ -0,0 +1,117 @@
+    echo iptables-persistent iptables-persistent/autosave_v4 boolean true | debconf-set-selections
+    echo iptables-persistent iptables-persistent/autosave_v6 boolean false | debconf-set-selections
+    apt-get install -y iptables-persistent
+    iptables -A INPUT -p tcp -m tcp --dport 80 -j ACCEPT
+    iptables -A INPUT -p tcp -m tcp --dport 443 -j ACCEPT
+    iptables -A INPUT -p tcp -m tcp --dport 3000 -j ACCEPT
+    iptables -A INPUT -p tcp -m tcp --dport 8080 -j ACCEPT
+    iptables -A INPUT -p tcp -m tcp --dport 2222 -j ACCEPT
+
+    iptables-save > /etc/iptables/rules.v4
+
+    /etc/init.d/netfilter-persistent reload
+
+  SHELL
+
+  ## USING RUBY ENV ###
+  config.vm.provision 'shell', privileged: false, inline: <<-SHELL
+    # download rbenv if the directory is missing
+    if cd ~/.rbenv && git rev-parse --verify --short=26 HEAD | grep -q $RBENV_VERSION; then
+      echo 'rbenv already downloaded at expected revision'
+    else
+      git clone https://github.com/sstephenson/rbenv.git ~/.rbenv
+      cd ~/.rbenv && git checkout $RBENV_VERSION
+    fi
+    # download ruby-build if the directory is missing
+    if cd ~/.rbenv/plugins/ruby-build && git rev-parse --verify --short=26 HEAD | grep -q $RBBUILD_VERSION; then
+      echo 'ruby-build already downloaded at expected revision'
+    else
+      git clone https://github.com/sstephenson/ruby-build.git ~/.rbenv/plugins/ruby-build
+      cd ~/.rbenv/plugins/ruby-build && git checkout $RBBUILD_VERSION
+    fi
+
+    RBENV_PATH='export PATH="$HOME/.rbenv/bin:$PATH"'
+    if grep -q "$RBENV_PATH" ~/.bash_profile ; then
+      echo 'rbenv path already in .bash_profile'
+    else
+      echo $RBENV_PATH >> ~/.bash_profile
+    fi
+
+    RBENV_INIT='eval "$(rbenv init -)"'
+    if grep -q "$RBENV_INIT" ~/.bash_profile ; then
+      echo 'rbenv init already in .bash_profile'
+    else
+      echo $RBENV_INIT >> ~/.bash_profile
+    fi
+  SHELL
+
+  config.vm.provision 'shell', privileged: false, inline: <<-SHELL
+   rbenv install 2.3.4
+   rbenv global 2.3.4
+  SHELL
+
+
+
+  config.vm.provision 'shell', privileged: false, inline: <<-SHELL
+   echo "gem: --no-document" > ~/.gemrc
+   gem install bundler
+
+   echo 'export RAILS_ENV=development' >> ~/.bash_profile
+   echo 'cd /vagrant' >> ~/.bash_profile
+
+   cd /vagrant
+   cp .env.example .env
+   sed -i 's/DATABASE_PASSWORD=""/DATABASE_PASSWORD="root"/g' .env
+
+   bundle install
+   RAILS_ENV=development bundle exec rake db:create
+   RAILS_ENV=development bundle exec rake db:migrate
+   RAILS_ENV=development bundle exec rake db:schema:load
+   RAILS_ENV=development bundle exec rake db:seed
+
+   echo 'alias huginn-server="rails server -b 0.0.0.0"' >> ~/.bash_profile
+   echo 'alias huginn-worker="nohup bundle exec rails runner bin/threaded.rb > tmp/worker.log &"' >> ~/.bash_profile
+
+   echo "-------"
+   echo "Use: '$: huginn-worker' command to start the worker in the background with output redirected to tmp/worker.log"
```

Comment:
Forgot to mention it earlier, we could use `bundle exec foreman start` here which would allow the user to specify the wanted process setup in the `Procfile`.